ouroboros-consensus/ouroboros-consensus-test-infra/src/Test/ThreadNet/Network.hs

```diff
@@ -54,6 +54,8 @@ import qualified Data.Set as Set
 import qualified Data.Typeable as Typeable
 import           Data.Void (Void)
 import           GHC.Stack
+import           System.Random (mkStdGen)
+import qualified Test.QuickCheck.Exception as QC
```

Comment:
Is this even used?

Reply:
No, I suspect it was a missmerge during rebase to master.